storage/Filesystems/FilesystemImpl.h

```diff
@@ -252,6 +253,32 @@ namespace storage
 
     }
 
+
+    class EnsureMounted : boost::noncopyable
+    {
+
+    public:
+
+	/**
+	 * Ensures that the filesystem is mounted somewhere.
+	 *
+	 * The mode is not enforced.
```

Comment:
What does this mean? A "TODO, not implemented"? But we do pass `--read-only` to `mount`.